SomyaTyagiji/Day1/01.c

```diff
@@ -0,0 +1,34 @@
+#include<stdio.h>
+int main()
+{ 
+    char name[50];
+    printf("what is your name? \n");
+    scanf("%s", name);
+    printf("welcome %s\n", name);
+  
+  char answer2[200]; 
+  printf("why do you want to learn c programming? \n");
+    scanf("%s", answer2);
```

Comment:
%s character will only print a single word. Try to pass whole statement as input and test code, it will give error.  Made necessary changes in your code.